compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -412,6 +412,15 @@ def ExportBenchmarkFuncsPass :
   ];
 }
 
+def FuseEncodingOpsIntoDispatchRegionsPass :
+    InterfacePass<"iree-flow-fuse-encoding-ops-into-dispatch-regions-pass", "mlir::FunctionOpInterface"> {
+  let summary = "Fuses encoding ops into dispatch regions, or forms new dispatches around them.";
```

Comment:
I think we can improve the summary slightly if we put `producer` to it. E.g., `Fuses encoding ops into producer dispatch regions, or ...`